app/views/home/_vertical_menu.html.erb

```diff
@@ -15,8 +15,9 @@
         <% when Menu::Item %>
           <%= menu_secondary_item item %>
         <% when Menu::Divider %>
-          <%= content_tag(:div, "", :class => "divider") %>
-          <%= content_tag(:span, _(item.caption), :class => "nav-item-pf-header") if item.caption %>
+          <%= content_tag(:li, :class => "list-group-item") do %>
+            <%= content_tag(:span, menu_title, :class => "vertical-sub-header") if item.caption %></a>
```

Comment:
menu_title should probably be item.caption, otherwise I'm getting wrong headers, see 
![screenshot_20171212_123301](https://user-images.githubusercontent.com/109773/33882501-9f5ca5f8-df38-11e7-93e9-9fe342e787bb.png)